setup.py

```diff
@@ -22,7 +22,7 @@
 # http://pypi.python.org/pypi/setuptools
 
 REQUIRES = ["urllib3 >= 1.15", "six >= 1.10", "certifi",
-            "python-dateutil", "marquez-python-codegen"]
+            "python-dateutil", "requests", "marquez-python-codegen"]
```

Comment:
As long as we're working on this file, please set the version of `marquez-python-codegen`.

Reply:
sure, what version should I point it to?

Reply:
I think the one you’re using now? Pip freeze should tell you for sure but it looks like 0.1.6.

Reply:
done